packages/tocco-ui/src/Panel/Panel.js

```diff
@@ -65,7 +74,15 @@ Panel.propTypes = {
   /**
    * Boolean to control if body can be opened or closed. Default value is 'true'.
    */
-  isToggleable: PropTypes.bool
+  isToggleable: PropTypes.bool,
+  /**
+   * Function gets passed from PanelGroup to ensure that only one Panel is open at one time.
+   */
+  toggleOpenStateAsGroup: PropTypes.func,
```

Comment:
the Panel itself actually shouldn't have to care about a surrounding group. Therefore, I'd simply rename the handler to `onToggle` and pass `true` to the event handler, if the panel is open or `false`, if closed.

`panelId` is not needed either.

Also see other comment in PanelGroup.js.